docs/nuxt.config.js

```diff
@@ -3,5 +3,10 @@ import theme from "@nuxt/content-theme-docs";
 export default theme({
   router: {
     base: '/vue-api-query/'
+  },
+  hooks: {
+    'vue-renderer:ssr:templateParams': function(params) {
+      params.HEAD = params.HEAD.replace('<base href="/vue-api-query/">', '')
+    }
```

Comment:
@JoaoPedroAS51 Mabye some comment here pointing into nuxt issue, for future final fixing.

Reply:
Good idea!

Reply:
Just note that the nuxt issue I mentioned is currently closed.